dockers/docker-fpm-frr/frr/zebra/zebra.conf.j2

```diff
@@ -7,6 +7,11 @@
 {% endblock banner %}
 !
 {% block fpm %}
+{% if (DEVICE_METADATA is defined) and ('localhost' in DEVICE_METADATA) and ('subtype' in DEVICE_METADATA['localhost']) and (DEVICE_METADATA['localhost']['subtype'].lower() == 'dualtor') %}
+! Disable next hop group support for DualTor
+no zebra nexthop kernel enable
```

Comment:
not sure what is the implication of this command. What do you mean disable next hop group support? From the linked issues, i'm not clear how this can resolve the issue.

Reply:
If you are referring to nexthop group feature, afaik, it should be default disabled.